src/audio/crossover/crossover.c

```diff
@@ -48,7 +49,7 @@ static inline void crossover_free_config(struct sof_crossover_config **config)
 /**
  * \brief Reset the state of an LR4 filter.
  */
-static inline void crossover_reset_state_lr4(struct iir_state_df2t *lr4)
+inline void crossover_reset_state_lr4(struct iir_state_df2t *lr4)
 {
```

Comment:
This change doesn't seem to be directly related to multiband drc component. Can you please add it in a separate commit with proper explanation of why it is needed?

Reply:
Ditto, we should split the exposing of the APIs into another commit for clarity.

Reply:
Thanks for the suggestion. I split into different commits.